endpoints/v2/errors.py

```diff
@@ -144,6 +144,13 @@ def __init__(self, uploaded=None, max_allowed=None):
             )
 
 
+class QuotaExceeded(V2RegistryException):
+    def __init__(self):
+        super(QuotaExceeded, self).__init__(
+            "QuotaExceeded", "Quota has been exceed on namespace", {}
+        )
```

Comment:
I think as it is this will return a 400. I think a 403 or 422 is more appropriate for this use case.